### ISSUE TITLE
"char_greeting" not showing up (characters)

### ISSUE THREAD

**Issue by iChristGit** [oobabooga/text-generation-webui#574]
### Describe the bug

I am using WSL. --cai-chat

When using the example it used to work but after copying it to a new .json file and changing it up, the greetings example text wont show up.
And now it wont show up in Example.json as well.
I even tried taking a fresh /characters/Example.json from git and still not getting the greetings.

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

Try making a new Bot.json character with greetings.

### Logs

```shell
No Error Logs
```

### System Info

```shell
Windows11 WSL
```

**Comment by bmoconno** [comment]
I don't have the latest version installed right now, but it's possible your issue is the same as issue #360 

Try shutting down the server, and rename the <character>.json file to something new that you've never used before and/or delete the existing <character>_persistent.json from the /logs/ folder. Then start the server back up to see if your greeting shows up as expected.

This has been bugging me also, and if someone else doesn't fix it soon, I'll probably see if I can fix it myself.

**Comment by olihough86** [comment]
Yep there is a bug where the <character>_persistent.json seems to get broken, removing it does seem to get the greeting back

**Comment by YakuzaSuske** [comment]
Yhea im having an issue with it after this new update, sadly deleting the persistent log or changing the json names doesn't fix it for me.
